js/app.js

```diff
@@ -0,0 +1,46 @@
+$(function(){
+
+// print background based on text entered.
+// text must be the name of a city.
+// city name has a class matching that city name in css. 
+// reset background with new submission.
+ 
+// listen for user clicking 'submit'/submitting text
+$("#submit-btn").click(changeBackground);
+ 
+function changeBackground(event) {
+ 
+  event.preventDefault();
+ 
+  //text entry - basically tells js to read what is in the input box (class of 'city-type')
+  var city = $("#city-type").val();
+  
+  //match entry with proper css class (name of city)
+  
+  if(city === "NYC" || city === "new york city" || city === "new york" || city === "New York" || city === "New York City" || city === "nyc"){
```

Comment:
@jcooper8190 good look to cover the cover the lower and uppercased values on entry. You could have also used `toUpperCase` and `toLowerCase` to have the conditional case match the case of the the inputed values.